README.md

```diff
@@ -15,11 +15,12 @@ Base Image [Debian:bullseye-slim](https://hub.docker.com/_/debian)
 | Dart                | Y                                                                                    | Y                                                                                            | Y                                                                                         | Y                                                                                           | Y                                                                                          | Y                                                                                                  | Y                                                                                               | Y                                                                                                 |
 | Node.js             | Y                                                                                    | Y                                                                                            | Y                                                                                         | Y                                                                                           | Y                                                                                          | Y                                                                                                  | Y                                                                                               | Y                                                                                                 |
+| Powershell          | Y                                                                                    | Y                                                                                            | Y                                                                                         | Y                                                                                           | Y                                                                                          | Y                                                                                                  | Y                                                                                               | Y                                                                                                 |
```

Comment:
Wasn't sure whether to call this Powershell 7 to differentiate it from Windows Powershell or Powershell Core. But I think Powershell 7 supercedes the others so I think it is somewhat impled.

Reply:
Yep. That's fine.